library/include/hiptensor/hiptensor_types.hpp

```diff
@@ -104,10 +104,20 @@ typedef enum
     //! Identity operator
     HIPTENSOR_OP_IDENTITY = 1,
     //! Square root operator
-    HIPTENSOR_OP_SQRT = 3,
+    HIPTENSOR_OP_SQRT = 2,
```

Comment:
Need to ensure these match the nv ones for api parity

Reply:
Double checked. Value 2 is consistant with cuda. 

<img width="661" alt="image" src="https://github.com/ROCm/hipTensor/assets/142121551/fe502bd4-3af5-449f-a934-6fb3133c08dd">